### ISSUE TITLE
npx create-expo-app@latest issues

### ISSUE THREAD

**Issue by leegee** [expo/expo#31628]
### Summary

```
lee@20240923-11:18:01:/xxx/norge-ufo$ npx create-expo-app@latest
√ What is your app named? ... ufo-norge
















✔ Downloaded and extracted project files.
> npm install
npm warn deprecated osenv@0.1.5: This package is no longer supported.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @babel/plugin-proposal-optional-catch-binding@7.18.6: This proposal has been merged to the ECMAScript standard and thus 
this plugin is no longer maintained. Please use @babel/plugin-transform-optional-catch-binding instead.
npm warn deprecated @babel/plugin-proposal-numeric-separator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this 
plugin is no longer maintained. Please use @babel/plugin-transform-numeric-separator instead.
npm warn deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and 
thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
npm warn deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-logical-assignment-operators@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-logical-assignment-operators instead.
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this 
plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm warn deprecated @babel/plugin-proposal-async-generator-functions@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.       
npm warn deprecated domexception@4.0.0: Use your platform's native DOMException instead

added 1577 packages, and audited 1578 packages in 1m

147 packages are looking for funding
  run `npm fund` for details

3 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

✅ Your project is ready!

To run your project, navigate to the directory and run one of the following npm commands.

- cd ufo-norge
- npm run android
- npm run ios # you need to use macOS to build the iOS project - use the Expo app if you need to do iOS development without a Mac
- npm run web
lee@20240923-11:19:31:/xxx/norge-ufo$ npm run web
npm error Missing script: "web"
npm error
npm error To see a list of scripts, run:
npm error   npm run
npm error A complete log of this run can be found in: C:\Users\leege\AppData\Local\npm-cache\_logs\2024-09-23T09_19_46_418Z-debug-0.log
lee@20240923-11:19:46:/xxx/norge-ufo$ 
```

### What platform(s) does this occur on?

Web

### Environment

```text
lee@20240923-11:20:58:/s/src/react-native-ufo-map/HeatmapApp/norge-ufo$ npx expo-env-info
Need to install the following packages:
expo-env-info@1.2.0
Ok to proceed? (y) y


  expo-env-info 1.2.0 environment info:
    System:
      OS: Windows 10 10.0.19045
    Binaries:
      Node: 20.17.0 - H:\Program Files\nodejs\node.EXE
      Yarn: 1.22.22 - ~\AppData\Roaming\npm\yarn.CMD
      npm: 10.8.2 - H:\Program Files\nodejs\npm.CMD
```

### Minimal reproducible example

Please see above.

**Comment by leegee** [comment]
> Hi @leegee! Did you try this, before running `npm run web` (or any other command)?
> 
> ```
> To run your project, navigate to the directory and run one of the following npm commands.
> 
> - cd ufo-norge
> ```

Yes, I did, as in the console output included in the ticket. Or did I miss something...?

Thanks
Lee

**Comment by leegee** [comment]
Stupidly I had not.... Thank you very much for taking the time. 

Now it's just a list of deprecations.

```
lee@20240930-12:01:19:/s/src/react-native-expo-1$ npx create-expo-app@latest
√ What is your app named? ... ufo-test
✔ Downloaded and extracted project files.
> npm install
npm warn deprecated osenv@0.1.5: This package is no longer supported.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @babel/plugin-proposal-optional-catch-binding@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-catch-binding instead.
npm warn deprecated @babel/plugin-proposal-numeric-separator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-numeric-separator instead.
npm warn deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
npm warn deprecated @babel/plugin-proposal-logical-assignment-operators@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-logical-assignment-operators instead.
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm warn deprecated @babel/plugin-proposal-async-generator-functions@7.20.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.1.6: Glob versions prior to v9 are no longer supported
npm warn deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm warn deprecated domexception@4.0.0: Use your platform's native DOMException instead

added 1578 packages, and audited 1579 packages in 2m

147 packages are looking for funding
  run `npm fund` for details

3 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

✅ Your project is ready!

To run your project, navigate to the directory and run one of the following npm commands.

- cd ufo-test
- npm run android
- npm run ios # you need to use macOS to build the iOS project - use the Expo app if you need to do iOS development without a Mac
- npm run web
npm notice
npm notice New patch version of npm available! 10.8.2 -> 10.8.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.3
npm notice To update run: npm install -g npm@10.8.3
npm notice
lee@20240930-12:04:54:/s/src/react-native-expo-1$ cd ufo-test/
lee@20240930-12:05:16:/s/src/react-native-expo-1/ufo-test$ npm run web

> ufo-test@1.0.0 web
> expo start --web

Starting project at S:\src\react-native-expo-1\ufo-test
Starting Metro Bundler
Networking has been disabled
Unable to reach well-known versions endpoint. Using local dependency map expo/bundledNativeModules.json for version validation
Dependency validation is unreliable in offline-mode
▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
█ ▄▄▄▄▄ █▄▄▄ ▀ ▀█ █ ▄▄▄▄▄ █
█ █   █ ██▄▀ █ ▀▄▄█ █   █ █
█ █▄▄▄█ ██▀▄ ▄▄██▀█ █▄▄▄█ █
█▄▄▄▄▄▄▄█ ▀▄█ ▀ ▀ █▄▄▄▄▄▄▄█
█ ▄█▄ █▄▀▀▄▀█▄▀█▀ █▄█▀█▀▀▄█
█   ▀█ ▄ █▄██▄▄▄▄▀▀███▄▀▀ █
█ ▄██▄ ▄    █▀█▄ █ ▄▀▀█▀ ██
█ ▄ ██▄▄██▀██▀▄▀ ▄▀ ██▄▀  █
█▄███▄█▄▄ █▀ ▄▄ █ ▄▄▄  ▄▀▄█
█ ▄▄▄▄▄ ██▀▀▀▄  █ █▄█ ███ █
█ █   █ █ ▄▄▄ ▀█▄ ▄  ▄ █▀▀█
█ █▄▄▄█ █▀ ▀ ▀█▄ ▄█▀▀▄█   █
█▄▄▄▄▄▄▄█▄█▄▄██▄▄▄▄█▄▄███▄█

› Metro waiting on exp://192.168.0.108:8081
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Web is waiting on http://localhost:8081

› Using Expo Go
› Press s │ switch to development build

› Press a │ open Android
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu
› Press o │ open project code in your editor

› Press ? │ show all commands

Logs for your project will appear below. Press Ctrl+C to exit.
Web Bundled 19017ms S:\src\react-native-expo-1\ufo-test\node_modules\expo-router\entry.js (842 modules)
λ Bundled 19162ms S:\src\react-native-expo-1\ufo-test\node_modules\expo-router\node\render.js (846 modules)
λ  WARN  props.pointerEvents is deprecated. Use style.pointerEvents 
  console.warn (S:\src\react-native-expo-1\ufo-test\node_modules\@expo\cli\src\start\server\serverLogLikeMetro.ts:91:21)
  warnOnce (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\modules\warnOnce\index.js:24:13)
  createDOMProps (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\modules\createDOMProps\index.js:534:13)
  createElement (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\exports\createElement\index.js:21:32)
  Component (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\exports\View\index.js:109:23)
  renderWithHooks (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:5662:16)
  renderForwardRef (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:5846:18)
  renderElement (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6009:11)
  renderNodeDestructiveImpl (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6108:11)
  renderNodeDestructive (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6080:14)
λ  WARN  accessibilityRole is deprecated. Use role. 
  console.warn (S:\src\react-native-expo-1\ufo-test\node_modules\@expo\cli\src\start\server\serverLogLikeMetro.ts:91:21)
  warnOnce (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\modules\warnOnce\index.js:24:13)
  createDOMProps (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\modules\createDOMProps\index.js:403:13)
  createElement (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\exports\createElement\index.js:21:32)
  Component (S:\src\react-native-expo-1\ufo-test\node_modules\react-native-web\dist\exports\View\index.js:109:23)
  renderWithHooks (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:5662:16)
  renderForwardRef (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:5846:18)
  renderElement (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6009:11)
  renderNodeDestructiveImpl (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6108:11)
  renderNodeDestructive (S:\src\react-native-expo-1\ufo-test\node_modules\react-dom\cjs\react-dom-server-legacy.node.development.js:6080:14)
Web Bundled 7397ms S:\src\react-native-expo-1\ufo-test\node_modules\expo-router\entry.js (841 modules)
 LOG  [web] Logs will appear in the browser console
```

**Comment by aabmar** [comment]
Both Expo 51 and Expo 52 has "props.pointerEvents is deprecated" warnings in the default template.

To recreate:

npx create-expo-app testapp
cd testapp
npm run web

In your browser, open the console. There you will find the warning "hook.js:608 props.pointerEvents is deprecated. Use style.pointerEvents Error Component Stack"
